lib/protocol/networks.js

```diff
@@ -864,9 +864,7 @@ const simnet = {};
 
 simnet.type = 'simnet';
 
-simnet.seeds = [
-  'aorsxa4ylaacshipyjkfbvzfkh3jhh4yowtoqdt64nzemqtiw2whk@127.0.0.1'
-];
+simnet.seeds = [];
 
```

Comment:
I can add seeder nodes here if anybody wants to start up a node and post their key:ip. You will need to restart and pull the commits that add in the keys.

Reply:
`al5pxjw6a6xirdzok3unbatjlsrszwx4bgumteoexdmihdsvpdphy@138.68.61.31`

Reply:
Kyokan seed nodes:

- `apmv724pysb6ifiqsqzrixg23dybnwehinvmzdv6v6n4h7w2da5rq@45.55.111.156`
- `anhaoy6kcptjt25qeas3zgmsrt5twwjlvqqa4pwnt6elsceazsez2@174.138.111.111`

Reply:
ajd6iidvt7p22eomfkzxtslgsrp2wllef3j4xqemvbknjfquppdmk@44.229.138.206